docs/freebsd.md

```diff
@@ -0,0 +1,54 @@
+# FreeBSD Install
```

Comment:
the first header should match the file name. (The header can be capitalized as wished)